src/graphql/config.js

```diff
@@ -10,7 +10,7 @@ exports.config = {
   metadata: {
     __name: 'metadata',
     document: './queries/metadata.graphql',
-    schema: { method: 'GET', url: 'https://metadata.p.rainbow.me/v1/graph' },
+    schema: { method: 'POST', url: 'https://metadata.p.rainbow.me/v1/graph' },
```

Comment:
afaik we should be using POST for all metadata calls, if you need to use POST feel free to rename the simulation config to something more generic and use that for points

iirc POST messes with cloudflare caching so we should only use where necessary